### PR TITLE
per account remember passphrase

### PR DESCRIPTION
**Pull request by heronhaye**

### CODE REVIEW

**Comment by heronhaye** [comment]
cc @patrickxb

File: go/libkb/config.go [hunk 1]
```diff
@@ -549,7 +549,18 @@ func (f *JSONConfigFile) GetVDebugSetting() string {
 func (f *JSONConfigFile) GetAutoFork() (bool, bool) {
 	return f.GetTopLevelBool("auto_fork")
 }
-func (f *JSONConfigFile) GetRememberPassphrase() (bool, bool) {
+func (f *JSONConfigFile) GetRememberPassphrase(username NormalizedUsername) (bool, bool) {
+	if username.IsNil() {
+		goto Fallback
```
Comment (jakob223) [inline go/libkb/config.go]: this goto is basically being an if, can you rearrange this?

Comment (heronhaye) [reply go/libkb/config.go]: One day I will find a legitimate use case

**Comment by heronhaye** [comment]
merging over flake

**Pull request merged**